### PR TITLE
Setup image promoter for k8s.io/autoscaling components

### PR DESCRIPTION
**Pull request by losipiuk**

### CODE REVIEW

**Comment by losipiuk** [comment]
@MaciekPytel @bskiba Please do the first pass review.

File: groups/groups.yaml [hunk 1]
```diff
@@ -478,6 +478,20 @@ groups:
...
+      ACL for staging k8s autoscaling components
+    settings:
+      ReconcileMembers: "true"
+    members:
+      - mwielgus@google.com
+      - lukaszos@google.com
+      - bskiba@google.com
+      - maciekpytel@google.com
+      - vivek.bagade92@gmail.com
+      - aleksandram@google.com
```
Comment (bskiba) [inline groups/groups.yaml]: +ksiedlecki, jbartosik

File: k8s.gcr.io/images/k8s-staging-autoscaling/OWNERS [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- mwielgus
+- bskiba
+- maciekpytel
+- losipiuk
```
Comment (bskiba) [inline k8s.gcr.io/images/k8s-staging-autoscaling/OWNERS]: +ksiedlecki, jbartosik

File: k8s.gcr.io/manifests/k8s-staging-autoscaling/promoter-manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# google group for gcr.io/k8s-staging-cluster-api is k8s-infra-staging-cluster-api@kubernetes.io
```
Comment (bskiba) [inline k8s.gcr.io/manifests/k8s-staging-autoscaling/promoter-manifest.yaml]: Comment is copy paste (shouldn't be cluster-api)

File: k8s.gcr.io/images/k8s-staging-autoscaling/images.yaml [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+    "sha256:003f98d9f411ddfa6ff6d539196355e03ddd69fa4ed38c7ffb8fec6f729afe2d": ["1.1.2-r2"]  # Branches: ['master', 1.9, 1.8]
+    "sha256:791836c2a2471ecb23a975cf12835f2a35ada3f9d841f3f1a363b83eca99e2aa": ["1.1.2"]  # Branches: ['master', 1.9, 1.8]
+- name: cluster-proportional-autoscaler-arm
+  dmap:
+    "sha256:5dc36021546a5e81a7e6fa3c79624a4ad1ba59802739f775997b36f955046fce": ["1.7.1"]
+    "sha256:2a2b1f922861dbeb3bc9644d9fb1f42e2b2ab72fc92c1e758ca226d715fad114": ["1.6.0"]
+    "sha256:30ebd2b18e99d9c2f5b4136e38c6043c8c5c79ee6cd4827b427f623c72ac3390": ["1.5.0"]
+    "sha256:b47c34445d99635d312c5b93c7817a2f80a2cc6ab100bd5d4ab46287497e37dc": ["1.4.0"]
+    "sha256:ce21b837f64e4de2cca088d2f2318df15011cdff6b47ce416b0281040232508c": ["1.3.0"]
+- name: cluster-proportional-autoscaler-arm64
```
Comment (bskiba) [inline k8s.gcr.io/images/k8s-staging-autoscaling/images.yaml]: I know this says autoscaler, but this is not actually currently under sig-autoscaling, we should resolve this with owners.

Comment (losipiuk) [reply k8s.gcr.io/images/k8s-staging-autoscaling/images.yaml]: I dropped the images from this PR alltogether.

**Pull request merged**